overview/dai.md

```diff
@@ -0,0 +1,58 @@
+{% hint style="warning" %} This documentation describes certain core aspects of MakerDAO's protocol that could change over time. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+# Dai Stablecoin
+
+Dai is an ERC-20 token native to the Ethereum Blockchain. Dai is a stable, fully collateralized, and decentralized currency pegged to the value of the U.S. Dollar and kept stable through a framework of aligned financial incentives. 
+
+Dai is held in digital wallets and supported on Ethereum and other popular blockchains. Dai can be used in the same manner as any other cryptocurrency: It can be freely sent to others, used as payment for goods and services, be held as a hedge against market volatility, and more. All Dai transactions are publicly viewable.
+
+A combination of stable collateral and an excess of volatile collateral backs every Dai in circulation. Under normal circumstances, the value of the collateral backing Dai will always equal or exceed the value of the Dai in circulation. The types, amount, and value of collateral backing Dai are publicly visible on the Ethereum blockchain in real time.
+
+## Why Dai?
+
+A decentralized digital economy needs stable money in order to function. Dai is a price-stable asset that can be used as a medium of exchange, store of value, and unit of account.
+
+Dai enables a wide range of financial activities and applications that have previously been impracticable due to the volatility of legacy cryptocurrencies like Bitcoin.
+
+Dai is the world's first unbiased stable currency that does not discriminate. Any individual or business can realize the advantages of digital money. No one can forcefully seize nor freeze Dai.
+
+## How does Dai stay stable?
+
+Dai is not a hard-pegged currency, so it does not perfectly track the value of the U.S. dollar. Rather, it maintains a free-floating peg that experiences extremely low volatility against the U.S. dollar.
+
+Dai achieves stability through a combination of external market forces, economic incentive mechanisms, and policy tools controlled by MKR token holders. Many different market actors behaving in a self-interested manner contribute to its stability.
+
+There are four primary mechanisms Dai uses to remain stable.
+
+### Peg Stability Modules
+
+The Maker Protocol uses contracts called PSMs (Peg Stability Modules) that allow Dai to be directly redeemed for other cryptocurrencies pegged to the U.S. dollar.
+
+At any time, any user can receive minted Dai in exchange for these cryptocurrencies. If a Peg Stability Module contains sufficient stock, any user can destroy Dai in exchange for receiving these cryptocurrencies.
+
+This mechanism is the most effective at maintaining the Dai peg in the short term.
+
+For more information about the PSM, please refer to the [Maker Manual](https://manual.makerdao.com/module-index/module-psm).
+
+### Maker Vaults
```

Comment:
I don't think you want to put Vaults under "How does Dai stay stable". Vault fees yes, but the existence of vaults is more for the existence of Dai, not its stability